lib/t/t_opf_softlims.m

```diff
@@ -107,6 +107,28 @@ function t_opf_softlims(quiet)
         sdefault.(prop{:}).ub   = 1;
     end
 end
+%% generator ordering
+mpc = mpc0;
+mpc.bus(:,QD) = mpc.bus(:,QD)*2;
+mpc.gen = mpc.gen([3,1,2],:);
+mpc.gen(:,[QMAX,QMIN]) = [1.1,-1.1;2.2,-2.2;3.3,-3.3];
+mpc.gen(:,PMIN) = [1.1;2.2;3.3];
+mpc.gen = [mpc.gen(1,:); mpc.gen(3,:); mpc.gen(2:end,:)];
+mpc.gen(2,GEN_STATUS) = 0;
+mpc.gencost = [mpc.gencost(1,:); mpc.gencost(3,:); mpc.gencost(2:end,:)];
+
+mpc.softlims = sdefault;
+mpc.softlims.QMAX.idx = [1,3];
+
+t = 'generator ordering: ';
```

Comment:
Rather than adding a specific run for checking that the original limits end up in the results, I think I'd prefer adding these checks to each of the existing tests. That would make sure that none of the limits get messed up no matter what the softlims inputs are.

Reply:
I wrote a utility function that does this, so I have no problem adding it after each test.
One question though:
At the moment the generators are ordered [1,2,3] in the original case and are all online. That means that there is no change in the `gen` matrix during the `ext2int` call. As a result, the check would most likely pass....but also not reveal much. 
Is this sufficient, or do we need to make sure there is a change in the `gen` matrix each time? This would entail re-doing any `t_is` test relating to generation since they assume the current structure.

Reply:
Well, I suppose if you have the time switching the order of the original generators would result in a more useful test.

Reply:
...I was afraid you'd say that :) Ok, will do.

Reply:
Done. this file will require a substantial change once the data structure changes...but that's later.